src/main/models/court.ts

```diff
@@ -0,0 +1,16 @@
+import {Protocol} from 'puppeteer';
+import integer = Protocol.integer;
+
+export class Court {
```

Comment:
shouldn't this be interface rather than a class?
we are getting rid of puppeteer so you might need to look into alternative